app/Controller/PagesController.php

```diff
@@ -36,6 +37,49 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
+
+/**
+ * TooManyMethods
+*/
+    public function foo(  ){
+
+    }
+public function display1() {
+}
+public function display2() {
+}
+public function display3() {
+}
+public function display4() {
+}
+public function display5() {
+}
+public function display6() {
+}
+public function display7() {
+}
+public function display8() {
+}
+public function display9() {
+}
+public function display10() {
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected 4 spaces, found 0
* Missing function doc comment
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->